spec/koans.js

```diff
@@ -256,17 +295,26 @@ describe('the JavaScript language', () => {
         return internal(4, 4);
       }
 
-      //expect(external()).toBe();
+      expect(external()).toBe(9);
     });
 
     it('may return arrays that contains closures and so on', () => {
+
+      //--------------------------------
+      //Que pasa AQUIIIII
+      //--------------------------------
+
+
```

Comment:
Es muy parecida a la de _'may return arrays that contains functions and so on'_, tenéis que crear una función que retorna un array, que en la posición 0 tiene una función a la cual apuntamos y recibe un argumento que es un número y esta debe devolver un array que opere con ese número y de el valor que nos indica el expect.